### ISSUE TITLE
Review mode UI lacks complete keyboard navigation

### ISSUE THREAD

**Issue by darrensiegel** [Simon-Initiative/oli-torus#358]
User cannot tab into the list of warnings.

User cannot use the arrow keys to switch between the selected warning.
